app/controllers/graphql_controller.rb

```diff
@@ -13,7 +13,11 @@ def execute
       # current_user: current_user,
     }
     result = StreetArtWalkBeSchema.execute(query, variables: variables, context: context, operation_name: operation_name)
-    render json: result
+    if result.to_h["data"].keys[0] == "tours"
+      redirect_to "/tours"
```

Comment:
This is Michael's clever workaround, which redirects to a restful route so we can handle the tours in good 'ole rails and escape from the hellscape of GraphQL

Reply:
![](https://media4.giphy.com/media/PmpA5ohOUl1xC/giphy.gif)

Reply:
nice!